app/src/main/java/org/mozilla/focus/utils/OkHttpWrapper.kt

```diff
@@ -13,6 +13,12 @@ object OkHttpWrapper {
 
     @JvmStatic
     fun onLowMemory() {
-        try { client.cache().evictAll() } catch (_: IOException) { /* We don't care. */ }
+        try { client.cache()!!.evictAll() }
```

Comment:
We can write more correct, concise code with null chaining. Check out the docs to figure it out :):  https://kotlinlang.org/docs/reference/null-safety.html Let me know if you have questions.

In general, it's bad to execute unsafe calls like `!!`. You really only want to use `!!` when you're sure the value will never be null. And even then, you generally want to avoid it because if the code changes, your assumptions about the value never being null can be invalidated and the program can crash.

Reply:
Please review https://github.com/psymoon/firefox-tv/commit/905d4996d6fa17d547ad3ff09d092789b2520475